src/Presentation/ConfigEditPageTextBuilder.php

```diff
@@ -45,7 +45,8 @@ public function createBottomHtml(): string {
 				'msg-wikibase-faceted-search-config-tab-name-missing' => $this->context->msg( 'wikibase-faceted-search-config-tab-name-missing' )->plain(),
 				'msg-wikibase-faceted-search-config-help-example' => $this->context->msg( 'wikibase-faceted-search-config-help-example' )->escaped(),
 				'exampleContents' => $this->getExampleContents(),
-				'array-itemTypes' => $this->getItemTypesData()
+				'array-itemTypes' => $this->getItemTypesData(),
+				'searchUrl' => $this->getSearchUrl(),
```

Comment:
I did not use a message string here for the full link, because the rest of the documentation is not translatable. It would look strange to have individual sentences in another language while the rest of the documentation cannot be translated. If we decide to make the documentation translatable then this can go away.